src/faebryk/exporters/pcb/kicad/pcb.py

```diff
@@ -319,83 +333,89 @@ def get_property_uuid(
 
         x, y = 0, 0
         current_prefix = None
-        for comp_name in _sort_by_address_prefix(nl_comps, comps_added):
-            comp = nl_comps[comp_name]
-            address_prefix = _get_address_prefix(comp)
-            footprint_identifier = comp.footprint
-            footprint = _get_footprint(footprint_identifier, fp_lib_path)
-            pads = {
-                p.pin: n
-                for n in nl_nets.values()
-                for p in n.nodes
-                if p.ref == comp_name
-            }
-
-            # Fill in variables
-            footprint.propertys["Reference"].value = comp_name
-            footprint.propertys["Value"].value = comp.value
-            footprint.propertys["atopile_address"] = fill_fp_property(
-                fp=footprint,
-                property_name="atopile_address",
-                layer="User.9",
-                value=get_property_value(comp, "atopile_address", "No atopile_address"),
-                uuid=get_property_uuid(comp, "atopile_address"),
-            )
-            footprint.propertys["LCSC"] = fill_fp_property(
-                fp=footprint,
-                property_name="LCSC",
-                layer="User.9",
-                value=get_property_value(comp, "LCSC", "No LCSC number"),
-                uuid=get_property_uuid(comp, "LCSC"),
-            )
-
-            if current_prefix is not None and address_prefix != current_prefix:
-                # separate prefix groups horizontally
-                x += HORIZONTAL_SPACING
-                y = 0
-
-            current_prefix = address_prefix
-
-            at = C_xyr(x=x, y=y, r=0)
-
-            # separate components vertically within group
-            y += VERTICAL_SPACING
-
-            if comp_name in comps_changed:
-                # TODO also need to do geo rotations and stuff
-                at = comps_changed[comp_name].at
+        for err_collector, comp_name in iter_through_errors(
```

Comment:
Dunno why the diff aligns so poorly. This should all just be indented one to add the error collection context. Double checking now

Reply:
Yep, confirmed there are no other changes